app/controllers/simulations_controller.rb

```diff
@@ -7,15 +7,21 @@ def index
 
   def show
     @simulation = Simulation.find(params[:id])
+    @matches = @simulation.simulation_match_presets
   end
 
   def new
     @simulation = Simulation.new
+    @matches = Match.pending.all
   end
 
   def create
+    matches = params[:matches]
     @simulation = Simulation.new(simulation_params)
     if @simulation.save
+      matches.each do |match_id, result|
```

Comment:
@csalvato, I created a view where the user can pre-select future results.

It took me some time to think how to get the data connected to params.